addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -44,15 +48,19 @@
                 </button>
                 <div class="dropdown-menu">
                     <t t-foreach="countries" t-as="country" t-if="country['country_id']">
-                        <a t-attf-href="/partners#{ current_grade and '/grade/%s' % slug(current_grade) or ''}#{country['country_id'][0] and '/country/%s' % slug(country['country_id']) or '' }#{ '?' + (search_path or '') + (country['country_id'][0] == 0 and '&amp;country_all=True' or '')}"
-                        class="dropdown-item" t-out="country['country_id'][1]"/>
+                        <a t-attf-href="/partners#{ current_grade and '/grade/%s' % slug(current_grade) or ''}#{country['country_id'][0] and '/country/%s' % slug(country['country_id']) or '' }?#{ keep_query('search', (country['country_id'][0] == 0 and 'country_all' or ''), 'industry')}"
+                        class="dropdown-item d-flex justify-content-between">
+                            <t t-out="country['country_id'][1]"/>&amp;nbsp;&amp;nbsp;
+                            <span class="badge text-white bg-secondary rounded-pill px-2 py-1" t-esc="country['country_id_count']"/>
```

Comment:
```suggestion
                            <span class="badge text-white bg-secondary rounded-pill px-2 py-1" t-esc="country['country_id_count'] or ''"/>
```
?

Reply:
You would like `''` instead of `0` ? not sure

Reply:
Indeed 0 is better. But why is it different for grades, then (line38)?

Reply:
It comes from a group by, so btw you should never have 0 :thinking:  but yes not coherent in the other (l38) to have the fallback on `or ''` 

E.G. no gold partner in Zimbabwe...
![image](https://github.com/user-attachments/assets/193e69ef-253c-41ef-9b25-55eb6d09e032)
![image](https://github.com/user-attachments/assets/48d3781b-5c2f-4e4d-80ff-34eb0ff8e39b)